database/reactors_pris_2016.csv

```diff
@@ -353,10 +358,12 @@ Pakistan,CHASNUPP-4,PWR,315,Under Construction,PAEC,12/18/2011,2011,6/1/2017,7/1
 Philippines,BNPP-1,PWR,620,Cancelled Constr.,RPNPC,7/1/1976,1976,,,,5/1/1986,,,,,
 Poland,ZARNOWIEC-1,PWR,440,Cancelled Constr.,EJZ,1/1/1983,1983,,,,9/4/1990,,,,,
 Poland,ZARNOWIEC-2,PWR,440,Cancelled Constr.,EJZ,1/1/1983,1983,,,,9/4/1990,,,,,
+Poland,CHOCZEWO,N/A,3000,Planned,,01/01/2029,2029,01/01/2029,01/01/2029,01/01/2029,,,,,,
+Poland,NONAME,N/A,3000,Planned,,01/01/2035,2035,01/01/2035,01/01/2035,01/01/2035,,,,,,
 Romania,CERNAVODA-1,PHWR,650,Operational,SNN,7/1/1982,1982,4/16/1996,7/11/1996,12/2/1996,,98.6,44.32234722,28.05728056,,
 Romania,CERNAVODA-2,PHWR,650,Operational,SNN,7/1/1983,1983,5/6/2007,8/7/2007,10/31/2007,,89.2,44.32234722,28.05728056,,
-Romania,CERNAVODA-3,PHWR,655,Suspended Constr.,SNN,2/9/1984,1984,,,,,,44.32234722,28.05728056,,
-Romania,CERNAVODA-4,PHWR,655,Suspended Constr.,SNN,8/15/1985,1985,,,,,,44.32234722,28.05728056,,
+Romania,CERNAVODA-3,PHWR,655,Suspended Constr.,SNN,2/9/1984,1984,01/01/2019,01/01/2019,01/01/2019,,,44.32234722,28.05728056,,
```

Comment:
Is this new data from pris? Why these additions to the raw data?

Reply:
These reactors are the planned reactors mentioned in the paper. They were previously listed as suspended construction without no expected date, but the dates were added so that the `cyclus-input-gen` would deploy them at that point.